app/helpers/application_helper.rb

```diff
@@ -2,4 +2,11 @@ module ApplicationHelper
   def hotwire_livereload_tags
     super if Rails.env.development?
   end
+
+  def round_to_lower_multiple_of_ten(number)
+    return number - 1 if number <= 10
+
+    modulo = number % 10
+    number - modulo
+  end
```

Comment:
We can use `Developer.count.round(-1)` instead of implementing this ourselves.

Reply:
Yeah, I also played around with it but didn't want to use it, because of these cases:
```
[5] pry(main)> 126.round(-1)
=> 130
```
It might be too much haha but that's the reason I didn't use it. 

Up to you though!

Reply:
I'm fine with that. If we have 195 developers and say "200+" that's a difference of 2.5% which I'm OK with. Thanks for checking though!

Reply:
Done! 👍